composeApp/src/commonTest/kotlin/com/arkivanov/minesweeper/game/DefaultGameComponentTest.kt

```diff
@@ -0,0 +1,25 @@
+package com.arkivanov.minesweeper.game
+
+import com.arkivanov.decompose.DefaultComponentContext
+import com.arkivanov.essenty.lifecycle.LifecycleRegistry
+import com.arkivanov.mvikotlin.main.store.DefaultStoreFactory
+import kotlinx.coroutines.test.StandardTestDispatcher
+import kotlinx.coroutines.test.TestCoroutineScheduler
+
+class DefaultGameComponentTest {
+    private val lifecycle = LifecycleRegistry()
+    private val componentContext = DefaultComponentContext(lifecycle = lifecycle)
+    private val storeFactory = DefaultStoreFactory()
```

Comment:
No need for `componentContext` and `storeFactory` properties, they can be just inlined.